libbeat/keystore/file_keystore.go

```diff
@@ -382,3 +386,11 @@ func (k *FileKeystore) checkPermissions(f string) error {
 func (k *FileKeystore) hashPassword(password, salt []byte) []byte {
 	return pbkdf2.Key(password, salt, iterationsCount, keyLength, sha512.New)
 }
+
+func (k *FileKeystore) validateKey(key string) error {
+	if strings.IndexAny(key, ".") != -1 {
+		return fmt.Errorf("invalid key format, key should not have '.', key: %s", key)
```

Comment:
I wonder if we should change this message to something like `is not supported yet`. I'm ok with not supporting it in the first iteration but I think moving forward to definitively have to (if possible) as it's kind of awkward having the dot nation in all our config files but not allow it for the key store. I think it's one of the first issues people will run into.

Reply:
I am ok with the "is not supported yet"

Reply:
@ruflin its possible just it require a bit more work on ucfg :)

Reply:
@ruflin Maybe  "Nested key is not supported yet, key: .."

Reply:
TBH I think `nested` will not make it very clear on what it's about. Already now I often have confusion around nested fields and nested documents etc in ES so I would stear away from that word. 


```
+		return fmt.Errorf("invalid key format. '.' in keys are not supported yet. key: %s", key)
 ```